documentation/sphinx/source/command-line-interface.rst

```diff
@@ -237,11 +250,43 @@ For each IP address or IP:port pair in ``<ADDRESS...>``, the command removes any
 
 For information on adding machines to a cluster, see :ref:`adding-machines-to-a-cluster`.
 
+kill
+----
+
+The ``kill`` command attempts to kill one or more processes in the cluster.
+
+``kill``
+
+With no arguments, ``kill`` populates the list of processes that can be killed. This must be run prior to running any other ``kill`` commands.
+
+``kill list``
+
+Displays all known processes. This is only useful when the database is unresponsive.
+
+``kill <ADDRESS>*``
+
+Attempts to kill all specified processes. Each address should include the IP and port of the process being targeted.
+
+``kill all``
+
+Attempts to kill all known processes in the cluster.
+
 lock
 ----
 
 The ``lock`` command locks the database with a randomly generated lockUID.
 
+maintenance
+-----------
+
+The ``maintenance`` command marks a particular zone ID (i.e. fault domain) as being under maintenance. A zone that is under maintenance will not have data moved away from it even if processes in that zone fail. This is useful when the amount of time that the processes in a fault domain are expected to be absent is reasonably short. The syntax for this command is ``maintenance [on|off] [ZONEID] [SECONDS]``.
+
+Running this command with no arguments will display the state of any current maintenance.
+
+Running ``maintenance on <ZONEID> <SECONDS>`` will turn maintenance on for the specified zone. A duration must be specified for the length of maintenance mode.
```

Comment:
I'm not sure if that's true or not. My understanding was that this won't cause any actual reduction in fault tolerance, but instead you would only see a reduction if a process is actually brought down. That's of course the same behavior whether you use maintenance mode or not, the only difference being that with maintenance mode you won't attempt to heal the cluster.

Reply:
Without maintenance mode, data on the failed process will be moved to other processes so that the data still has three replicas (i.e., on a three-member team). If the moving finishes within the maintenance window, we will have 3 SSes to serve the data.

With maintenance mode, the data will not be moved away. Within the maintenance window, we only have 2 SSes to serve the data, IIRC. Later, when the maintenance window ends, the process  on the maintained node will restart and catch up with the mutations in the tLog. Did I miss something? 

(Yeah, if we count the data on tLog, the replica factor does not decrease.)

Reply:
Right, but using maintenance mode doesn't change fault tolerance itself as far as I know. It just prevents data movement from activating to heal the cluster if you lose processes.

Reply:
I've added a clarifying comment to this effect in the docs.